integration-tests/src/tests/client/features/delegate_action.rs

```diff
@@ -826,7 +826,12 @@ fn meta_tx_create_eth_implicit_account_fails() {
 fn meta_tx_create_and_use_implicit_account(new_account: AccountId) {
     let relayer = bob_account();
     let sender = alice_account();
-    let node = RuntimeNode::new(&relayer);
+
+    let node = RuntimeNode::new_with_modified_config(&relayer, |runtime_config| {
+        // Increase the outgoing receipts limit to allow the large receipt to be processed immediately.
+        // Without this change the receipt would be processed somewhere in the next few blocks.
+        runtime_config.witness_config.outgoing_receipts_usual_size_limit = 200_000;
+    });
```

Comment:
nit: Again it feels like those should be part of the congestion control config and the `RuntimeConfigStore::test_congestion_control_disabled()` should maybe included that override.